packages/core/mesh/rpc/src/rpc.ts

```diff
@@ -170,16 +170,15 @@ export class RpcPeer {
     await Promise.race([this._remoteOpenTrigger.wait(), this._closingTrigger.wait()]);
 
     this._clearOpenInterval?.();
-    this._state = RpcState.OPENED;
```

Comment:
the next if-check could never trigger (`this._state !== RpcState.OPENED`). instead assign before waking up `_remoteOpenTrigger`

Reply:
lol